### ISSUE TITLE
Add setting to completely remove the Console footer from view

### ISSUE THREAD

**Issue by moniuch** [stackblitz/core#642]
(Based on experience in a TS Blank Project)

For some projects or fiddles the console is completely irrelevant, as you might want to only test css, rather than js/ts output. The expanding console footer actually gets disturbing, when what you want to test is flexbox heights on a page container, for example. In such cases it would be desirable to be able to completely remove the footer so that it does not consume any height, and user is confident as for what 100% height is.

**Comment by echonax** [comment]
There is hideDevTools: true, option. Did you try that?

**Comment by moniuch** [comment]
Nope. Where is it accessible from? Looked into Share menu as well as Settings (just below Dependencies) and haven't noticed anything like it.

**Comment by echonax** [comment]
You provide it via EmbedOptions.
Example:

```
    sdk.embedProject('myDiv', project,
    {
      height: 500,
      hideDevTools: true
    });
```

**Comment by moniuch** [comment]
Oh that's why. I wasn't embedding, just expected it somewhere to toggle
temporarily on demand

**Comment by echonax** [comment]
You provide it via EmbedOptions.
Example:
```
    sdk.embedProject('myDiv', project,
    {
      height: 500,
      hideDevTools: true
    }
);
```

**Comment by echonax** [comment]
Oh I assumed you were using embedding. If you can give more info about how you are using it we might be able to find another solution.

**Comment by moniuch** [comment]
I just open a stackblitz, to write some scroll related code, then realize
that my style and positioning (against bottom edge) might be affected by
the devtool opening widget, so I want to hide it quick, just for the sake
of styling.

Intuitively, I would look for it around the Settings on he left hand
sidebar.

Alternatively, we could have twoborientations for that widget: horizontal
(docked to bottom) and vertical (docked to right).

But I would prefer just hiding/showing it.
